ludwig/contribs/mlflow/__init__.py

```diff
@@ -28,7 +28,7 @@ def _get_or_create_experiment_id(experiment_name):
 
 
 class MlflowCallback(Callback):
-    def __init__(self, tracking_uri=None):
+    def __init__(self, tracking_uri=None, skip_artifact_logging: bool = False):
```

Comment:
I know there's precedent for naming like this, but typically it's easier to read a variable name like `log_artifacts = True` rather than `if not skip_artifact_logging`

Reply:
I agree.